scripts.txt

```diff
@@ -0,0 +1,58 @@
+# List of tools/scripts that can be called via command-line
```

Comment:
This list is in setup.py already. So long as we *install* the package before testing it (which we *should* be doing:

* https://github.com/pypa/packaging.python.org/issues/320
* https://blog.ionelmc.ro/2014/05/25/python-packaging/#the-structure
* https://hynek.me/articles/testing-packaging/

)

you can use `pkg_resources.get_entry_map('spinalcordtoolbox')['console_scripts'].keys()` below instead of loading this file.

Reply:
ps. there's also [`importlib.metadata`](https://docs.python.org/3/library/importlib.metadata.html) coming to the stdlib, but it doesn't have a stable API yet.